deployments/deploy.js

```diff
@@ -4,11 +4,12 @@ async function main() {
     const UsersInfo = await ethers.getContractFactory("UsersInfo");
     const user_info = await UsersInfo.deploy();
```

Comment:
The "Nice" console log statement does not provide any meaningful information and could be considered unnecessary. It would be better to remove it or replace it with a more informative message, as it does not provide any useful information for debugging or monitoring purposes.